src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResource.java

```diff
@@ -385,5 +385,9 @@ static String getResource(String key) {
 				{"R_kerberosLoginFailed", "Kerberos Login failed: {0} due to {1} ({2})"},
 				{"R_StoredProcedureNotFound", "Could not find stored procedure ''{0}''."},
 				{"R_jaasConfigurationNamePropertyDescription", "Login configuration file for Kerberos authentication."},
+				{"R_SQLVariantSupport", "sql_variant datatype is not supported in pre-SQL 2008 version"},
+                {"R_invalidProbbytes", "sql_variant: invalid probBytes for {0} type."},
+                {"R_invalidStringValue", "sql_variant does not support string values more than 8000"},
+
     };
```

Comment:
entry R_variantNotSupported is not used anywhere, we can delete it now. 
Also, error messages should start with Capital letter, maybe replace `sql_variant` with `SQL_VARIANT`? 
Also, error messages should end with period. Since you are working on this file, could you please also add period to error messages that missing period?